backend/src/v2/compiler/argocompiler/container.go

```diff
@@ -30,9 +30,9 @@ import (
 const (
 	volumeNameKFPLauncher    = "kfp-launcher"
 	volumeNameCABundle       = "ca-bundle"
-	DefaultLauncherImage     = "gcr.io/ml-pipeline/kfp-launcher@sha256:bef55a344574a25c557256d7c66cb19edacfd2008d694e5b6bb5b612d59feae0"
+	DefaultLauncherImage     = "ghcr.io/kubeflow/kfp-launcher:2.4.0"
```

Comment:
nitpick but would be nice to have SHAs here instead of labels, like the original image (useful for disconnected setups, etc)

Reply:
this is consistent with how we treat the other images, I agree sha's are great, but users will continue to have the option to specify sha's via env vars as they need

I think if we are going to switch to shas we should do it across the board, and I would prefer to do that as a separate initiative rather than introduce too many new variables for one release